shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -688,8 +688,8 @@ - (void)updateViewportMetrics {
 - (CGFloat)statusBarPadding {
   UIScreen* screen = self.view.window.screen;
   CGRect statusFrame = [UIApplication sharedApplication].statusBarFrame;
-  CGRect viewFrame =
-      [self.view convertRect:self.view.bounds toCoordinateSpace:screen.coordinateSpace];
+  CGRect viewFrame = [self.view convertRect:self.view.bounds
```

Comment:
Not clear what the formatting for .mm is, but looks like auto clang formatted?

Reply:
Yep this was auto-formatted by clang.